test/Frame.spec.jsx

```diff
@@ -300,12 +301,12 @@ describe('The Frame Component', () => {
 
     const component2 = ReactDOM.render(
       <ul className="container">
-        <li key="2">
+        <li>
           <Frame>
             <p>Text 2</p>
           </Frame>
         </li>
-        <li key="1">
+        <li>
```

Comment:
There was an issue with the last element not being populated when `key="1"` but if I switched this to `key="3"` or just removed the keys it would work as expected.